lib/code-manager.js

```diff
@@ -331,3 +331,40 @@ export function findCodeBlock(editor: atom$TextEditor) {
   }
   return [getRow(editor, row), row];
 }
+
+export function foldCurrentCell(editor: atom$TextEditor) {
+  const cellRange = getCurrentCell(editor);
+  const newRange = adjustCellFoldRange(editor, cellRange);
+  editor.setSelectedBufferRange(newRange);
+  const selection = editor.selections[0];
+  selection.fold();
+}
+
+export function foldAllButCurrentCell(editor: atom$TextEditor) {
+  const initialSelections = editor.getSelectedBufferRanges()
+
+  const allCellRanges = getCells(editor).slice(1);
+  const currentCellRange = getCurrentCell(editor);
+  const allButCurrentCell = allCellRanges.filter(item => !item.isEqual(currentCellRange))
+  var newRanges = [];
+  for (var i = 0; i < allButCurrentCell.length; i++) {
+    newRanges.push(adjustCellFoldRange(editor, allButCurrentCell[i]));
+  }
```

Comment:
This is totally valid javascript, but since you asked for some code style advise this could be rewritten as:

```js
const newRanges = allCellRanges
  .filter(cellRange => !cellRange.isEqual(currentCellRange))
  .map(cellRange => adjustCellFoldRange(editor, cellRange));
```

This is more concise and (arguable) more readable.

Reply:
Thanks! That does look better. I'll update the PR later.